devops/DeployCloudformationS3Bucket/deployStackS3Bucket.py

```diff
@@ -1,32 +1,54 @@
-#Deploys a Stack and S3 Bucket To Cloudformation.
+#Deploys a stack and S3bucket to cloudformation.
+
+#Imports Python libraries
 import boto3
 import re
 import os
 import sys
 import argparse
-
+import random
+import string
```

Comment:
Don't think you need to import the string library

Reply:
@eknigge Thanks for the feedback. I squashed most of the testing commits, so not so many now.

Reply:
@eknigge The import string library is required in code. Thanks.